abstract-factory/pom.xml

```diff
@@ -31,6 +31,11 @@
 			<artifactId>junit-jupiter-engine</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.projectlombok</groupId>
+			<artifactId>lombok</artifactId>
+			<version>1.18.12</version>
+		</dependency>
```

Comment:
I think Lombok dependency should be introduced at the parent `pom.xml` level and then referenced here. Related to issue https://github.com/iluwatar/java-design-patterns/issues/1503 which is under construction.